src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReferenceDiagnosticReport.java

```diff
@@ -0,0 +1,59 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Example RFF+SRI:13/CH001137K/211010191093'
+ */
+@EqualsAndHashCode(callSuper = false)
+@Builder
+@Getter
+@AllArgsConstructor
+public class ReferenceDiagnosticReport extends Segment {
+
+    private static final String KEY = "RFF";
+    private static final String QUALIFIER = "SRI";
+    private static final String KEY_QUALIFIER = KEY + "+" + QUALIFIER;
+
+    private final @NonNull String referenceNumber;
+
+    public static ReferenceDiagnosticReport fromString(final String edifactString) {
+        if (!edifactString.startsWith(KEY_QUALIFIER)) {
+            throw new IllegalArgumentException(
+                "Can't create " + ReferenceDiagnosticReport.class.getSimpleName() + " from " + edifactString);
+        }
+
+        String[] keySplit = Split.byPlus(edifactString);
+        String[] valueSplit = Split.byColon(keySplit[1]);
+
+        return new ReferenceDiagnosticReport(valueSplit[1]);
```

Comment:
[Optional] I feel it would be more readable if we created a variable, i.e. referenceNumber
```suggestion
        final String referenceNumber = Split.byColon(keySplit[1])[1];
        return new ReferenceDiagnosticReport(referenceNumber);
```

```suggestion
        final String referenceNumber = valueSplit[1];
        return new ReferenceDiagnosticReport(referenceNumber);
```